### PR TITLE
Alphabet last PR from me

### PR DESCRIPTION
**Pull request by h-2**
Just renaming and changing comparison operators to compare by value. But would be good to get in soon, because touches lots of files.

### CODE REVIEW

File: include/seqan3/alphabet/all.hpp [hunk 1]
```diff
@@ -123,7 +123,7 @@
  *
  * |                                  | seqan3::Semialphabet | seqan3::WritableSemialphabet | seqan3::Alphabet | seqan3::WritableAlphabet | Aux |
  * |----------------------------------|:--------------------:|:----------------------------:|:----------------:|:------------------------:|:---:|
- * | seqan3::alphabet_size_v          | ✅                    | ✅                            | ✅                | ✅                        |     |
+ * | seqan3::alphabet_size          | ✅                    | ✅                            | ✅                | ✅                        |     |
```
Comment (smehringer) [inline include/seqan3/alphabet/all.hpp]: indentation :nail_care:

File: test/unit/alphabet/composite/alphabet_variant_test.cpp [hunk 2]
```diff
@@ -250,19 +250,19 @@ TEST(alphabet_variant_test, rank_type)
     EXPECT_TRUE((std::is_same_v<alphabet3_t::rank_type, uint16_t>));
 }
 
-TEST(alphabet_variant_test, value_size)
+TEST(alphabet_variant_test, alphabet_size)
```
Comment (smehringer) [inline test/unit/alphabet/composite/alphabet_variant_test.cpp]:
```suggestion
TEST(alphabet_variant_test, alphabet_size_)
```
?

Comment (h-2) [reply test/unit/alphabet/composite/alphabet_variant_test.cpp]: ?

Comment (h-2) [reply test/unit/alphabet/composite/alphabet_variant_test.cpp]: I changed although I don't think it was necessary.

**Pull request merged**